test/common/testlib.py

```diff
@@ -824,6 +832,12 @@ def setUp(self):
                 machine.dhcp_server()
 
         if self.machine:
+            # Pages with debug enabled are huge and loading/executing them is heavy for browsers
+            # To make it easier for browsers and thus make tests quicker, disable packagekit and systemd preloads
+            # Only "TEST_OS_DEFAULT" has debug build enabled, see `build_and_install()` in `test/image-prepare`
+            if self.machine.image == testvm.TEST_OS_DEFAULT:
+                self.disable_preload("packagekit", "systemd")
```

Comment:
This also disabled the preloading of the Overview, which might or might not change the timing in tests in unexpected ways. Just wanted to point that out.

Reply:
Yes, disabling the preloading of the overview is really the main point here, as that gets loaded in most tests.